### PR TITLE
[Auengine] corrected extractions logic

### PR DESCRIPTION
**Pull request by Oteng**
The way the video download url was been extracted was
not working well so i change it for it to extract the
correct url

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ updates_key.pem
 test/testdata
 .tox
 youtube-dl.zsh
+.idea
```
Comment (jaimeMF) [inline .gitignore]: This is probably specific to your editor, you [should add them to your global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead.

**Comment by jaimeMF** [comment]
The test video is an episode [from an anime](http://en.wikipedia.org/wiki/The_Legend_of_the_Legendary_Heroes#Anime), I doubt they have the rights to distribute it.
We must find one video that doesn't infringe copyright, otherwise we'll have to remove the extractor.

File: .gitignore (hunk 1 above)
Comment (phihag) [inline .gitignore]: Oh well, doesn't hurt.

**Comment by phihag** [comment]
Thanks, merged.

As @jaimeMF mentioned, the extractor is danger of removal and will be removed within 10 days if a valid example URL cannot be found.

**Comment by Oteng** [comment]
about my editor sorry. About the extractor i am acctually working on a project based on this wonderfull project and in an attempted to get a passing build i decided to work on all the failing extractors and send you guys a pull request as a way of contributing back to the project that is helping me. Thanks guys

**Pull request merged**